include/graphics.h

```diff
@@ -20,15 +20,23 @@ along with TicTacToe.  If not, see <https://www.gnu.org/licenses/>.
 
 #if !defined(NXDK)
 #include <SDL2/SDL.h>
+#include <SDL2/SDL_ttf.h>
 #else
 #include <SDL.h>
+#include <SDL_ttf.h>
 #endif
 
 int windowWidth;
 int windowHeight;
+TTF_Font *smallFont;
```

Comment:
Not sure if your naming scheme is good? It can be used, sure, it's not wrong. But since you are only using them for specific reasons, why not give them proper names like "copyright font" "score font" etc.

Reply:
(unless you are thinking about reusing them)

Reply:
I am thinking about reusing at least one of them, will correct in a future PR if something is left unused.